src/test-interp.cc

```diff
@@ -718,6 +718,12 @@ TEST_F(InterpGCTest, Collect_DeepRecursion) {
   EXPECT_EQ(1u, store_.object_count());
 }
 
+TEST_F(InterpGCTest, Store_Assignment) {
```

Comment:
Maybe add a comment here.  Something like:

"Verify that move assignment operator works.  We previously has issues with double free that this test would fail under with ASAN".